### PR TITLE
Add `Kyoku: Original Song & Beatmapping Contest` news post

### PR DESCRIPTION
**Pull request by MChecaH**
[web preview](https://osu1.roanh.dev/home/news/2024-05-10-kyoku-original-song-and-beatmapping-contest), [#osu-news thread](https://discord.com/channels/188630481301012481/1237198052054274058)

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The hosts have proof read the post and reviewed it
- [x] contest page

### CODE REVIEW

File: news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Submission
+
+Submissions will be handled through the [contest’s Mappers’ Guild Page](https://mappersguild.com/contests/listing?contest=6616268c100297322b373789), and will be anonymized upon submission, including artist names. A livestream may be held on the day of results to show the rankings!
+
+## Prizes
+
+| Ranking | Prizes |
+| :-- | :-- |
+| Winning Team | $100USD + Profile Badge |
```
Comment (MChecaH) [inline news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md]: Potentially add contest points for the winning team, hyperlinked to https://osu.ppy.sh/wiki/en/Contests/Contest_points
```suggestion
| Winning Team | $100USD + Profile Badge + [Contest Points](/wiki/Contests/Contest_points) |
```

File: news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+| Judging | August 23 to August 31 |
+| Results | September 1 |
+
+## Song rules
+
+Starting June 10th, 2024, musicians will be allowed to send their song to any of the organisers to check over for approval, under these rules:
+
+- **Songs must be original works created for the contest**, or at the very least, previously unreleased. Anything found to be breaking this rule will be immediately disqualified.
+- **Songs must stay unreleased until the day of results.**
+- **Songs must be 2:00–6:00 in drain time.**
```
Comment (Walavouchey) [inline news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md:30]: drain time only makes sense for beatmaps. this is supposed to just mean song length, right?

**Review by MChecaH — COMMENTED** [review]
Couple of comments that the hosts should look over.

File: news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+- **All submissions must include a single difficulty.**
+- **Drain time in the song should be split between all mappers participating as evenly as possible.**
+- **Submissions must be in osu! mode only.**
+- **All maps must follow the ranking criteria.**
+- **Storyboards/videos are not permitted for the submission.** This is to prevent bias formed by the presentation of the map.
+
+## Submission
+
+Submissions will be handled through the [contest's Mappers' Guild page](https://mappersguild.com/contests/listing?contest=6616268c100297322b373789), and will be anonymised upon submission, including artist names. A livestream may be held on the day of results to show the rankings!
```
Comment (Walavouchey) [inline news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md:47]: should include information about song approval

**Review by Walavouchey — COMMENTED** [review]
based on the [info doc](https://docs.google.com/document/d/1DjLQN0wWzT25un9fUei9jmZqZdJ8Euvelf6Hgk0R4A0/view), this is missing judging criteria

File: news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+| Song production & mapping period | June 10 to August 22 |
+| Judging | August 23 to August 31 |
+| Results | September 1 |
+
+## Song rules
+
+Starting June 10th, 2024, musicians will be allowed to send their song to any of the organisers to check over for approval, under these rules:
+
+- **Songs must be original works created for the contest**, or at the very least, previously unreleased. Anything found to be breaking this rule will be immediately disqualified.
+- **Songs must stay unreleased until the day of results.**
```
Comment (Walavouchey) [inline news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md:29]: this probably applies to the map as well? that the map should not be uploaded via bss

File: news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+## Schedule
+
+| Phase | Dates |
+| --: | :-- |
+| Registrations | Now to June 9 |
+| Song production & mapping period | June 10 to August 22 |
+| Judging | August 23 to August 31 |
+| Results | September 1 |
+
+## Song rules
```
Comment (Walavouchey) [inline news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md:26]: based on a [q&a in the contest discord](https://discord.com/channels/1217150900498333766/1217156142434615438/1237711823449362473), you may want to add that the song can be updated after approval while the map is being made as long as the "fundamental composition isn't changed (as to screw up the map's rhythm choice)"

File: news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+## Submission
+
+Submission of the *final beatmap* will be handled through the [contest page](https://osu.ppy.sh/community/contests/211), and will be anonymised upon submission, including artist names.
+
+Only one team member needs to submit, and you can reupload your submission as many times as you want until the deadline of August 22!
+
+## Prizes
+
+A livestream may be held on the day of results to show the rankings!
```
Comment (Walavouchey) [inline news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md:55]: what is the twitch channel?

File: news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+| Phase | Dates |
+| --: | :-- |
+| Registrations | Now to June 9 |
+| Song production & mapping period | June 10 to August 22 |
+| Judging | August 23 to August 31 |
+| Results | September 1 |
+
+## Song rules
+
+Starting June 10th, 2024, musicians will be allowed to send their song to any of the organisers to check over for approval, under these rules:
```
Comment (Walavouchey) [inline news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md:28]: via discord? the organisers would be [bokeru](https://osu.ppy.sh/users/11308433), [chidodou](https://osu.ppy.sh/users/13868581), [Supportive](https://osu.ppy.sh/users/21482068) and [Whithard](https://osu.ppy.sh/users/24552396), right?

File: news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+- **Songs must be original works created for the contest**, or at the very least, previously unreleased. Anything found to be breaking this rule will be immediately disqualified.
+- **Songs must stay unreleased until the day of results.**
+- **Songs must be 2:00–6:00 in drain time.**
+- **Obvious bait/troll submissions will be immediately disqualified.**
+- **No commissions allowed.** This is to prevent creating an environment where people can outprice others.
+- **Teams cannot be smaller than 2.** Mappers are allowed to be producers too!
```
Comment (Walavouchey) [inline news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md:37]: this along with team size limits and the google form should be in a `## Registration` section imo

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**